sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClientOptions.cs

```diff
@@ -14,7 +14,7 @@ public class CommunicationIdentityClientOptions : ClientOptions
         /// <summary>
         /// The latest version of the identity service.
         /// </summary>
-        internal const ServiceVersion LatestVersion = ServiceVersion.V2021_03_07;
+        internal const ServiceVersion LatestVersion = ServiceVersion.V2021_02_22_preview1;
```

Comment:
Why is the latestVersion updated to V2021_02_22_preview1?

Reply:
Yeah the version should be a preview higher than the previous LatestVersion.

Reply:
Reverted the latest version to be V2021_03_07. I will abandon this PR and create a new one as I've rebased and the new iteration could show lot of unnecessary changes that got pulled in.